src/main/java/ch/njol/skript/classes/data/DefaultFunctions.java

```diff
@@ -466,11 +469,31 @@ public ColorRGB[] executeSimple(Object[][] params) {
 				Long green = (Long) params[1][0];
 				Long blue = (Long) params[2][0];
 				
-				return CollectionUtils.array(new ColorRGB(red.intValue(), green.intValue(), blue.intValue()));
+				return CollectionUtils.array(ColorRGB.fromRGB(red.intValue(), green.intValue(), blue.intValue()));
 			}
 		}).description("Returns a RGB color from the given red, green and blue parameters.")
 			.examples("dye player's leggings rgb(120, 30, 45)")
 			.since("2.5");
+
+		if (Skript.classExists("org.joml.Quaternionf"))
+			Functions.registerFunction(new SimpleJavaFunction<Quaternionf>("quaternionf", new Parameter[] {
```

Comment:
Can we get an Axis Angle function too?
eg: `axisAngle(vector(1,2,3), 90)`

Reply:
Ya, i'll add it to the todo list.